hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -690,6 +747,8 @@ private PreHandleResult addMissingSignatures(
         signatureExpander.expand(sigPairs, expanded);
         if (payerKey != null && !isHollow(payer)) {
             signatureExpander.expand(payerKey, sigPairs, expanded);
+        } else if (isHollow(payer)) {
+            context.requireSignatureForHollowAccount(payer);
         }
         signatureExpander.expand(currentRequiredPayerKeys, sigPairs, expanded);
```

Comment:
Just noticed there's a typo in this name `currentRequiredPayerKeys`---it's actually the required **non**-payer keys haha (c.f. [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java#L675)).

Reply:
haha yes 😄 Changed. Thank you!